tests/providers/test_ssn.py

```diff
@@ -873,3 +873,21 @@ class TestTlPh(TestEnPh):
     def setup_faker(self):
         self.fake = Faker('tl_PH')
         Faker.seed(0)
+
+
+class TestTrTr(unittest.TestCase):
+    num_sample_runs = 1000
+
+    def setup_factory(self):
+        self.factory = Faker('tr_TR')
```

Comment:
could you please follow the patters that other tests are using? Sepcifically:
* use `setUp` instead of `setup_factory`
* use `self.fake` instead of `self.factory`
* seed the faker with `Faker.seed(0)`

```python
class TestTrTr(unittest.TestCase):
    num_sample_runs = 1000
    def setUp(self):
        self.fake = Faker('tr_TR')
        Faker.seed(0)
```

Reply:
Okey, I got it.